zilliqa/src/api/erigon.rs

```diff
@@ -5,13 +5,13 @@ use jsonrpsee::{types::Params, RpcModule};
 
 use crate::node::Node;
 
-use super::types::EthBlock;
+use super::types::eth::Block;
 
 pub fn rpc_module(node: Arc<Mutex<Node>>) -> RpcModule<Arc<Mutex<Node>>> {
     super::declare_module!(node, [("erigon_getHeaderByNumber", get_header_by_number)])
 }
 
-fn get_header_by_number(params: Params, node: &Arc<Mutex<Node>>) -> Result<Option<EthBlock>> {
+fn get_header_by_number(params: Params, node: &Arc<Mutex<Node>>) -> Result<Option<Block>> {
```

Comment:
I like the explicit specification of `eth::Block` in the other files, perhaps we should keep it here just for visual clarity at a glance?

I like the idea of having a convention of specifying the namespace inline for any types which whose concepts might be shared between the two APIs (so blocks, logs, transactions and similar).

Reply:
Agreed, will change

Reply:
Done - I also did the same for the `Otterscan`-prefixed types in `eth.rs` and moved them to `ots.rs` instead.